app/discover/procurement/page.tsx

```diff
@@ -0,0 +1,361 @@
+</Text>
+
+<Text>
+With new functionalities, one should remember that they must be maintained. When you create new functionalities to your Oskari service, remember to inform NLS both beforehand and after the development has taken place. By contacting the NLS FI&apos;s Oskari technical team before you start the development, your developers can get best practices for the development process and further ideas on how to create the functionalities that you want. 
+</Text>
+
+<Text>
+When the new functionalities are in use, one should publish the code in a GitHub repository and inform the Oskari technical team - this helps them to take into account the dependencies of the new functionality when they are making new versions of Oskari. There might be dependencies, libraries or functionalities in the Oskari core that are affect the new functionality, and if you don’t inform the Oskari technical team about what you are doing or what you have done, it is possible that something gets broken when it&#39;s the time to update the Oskari instance next time.
+</Text>
+
+<Text>
+The changes into Oskari should never be done into the core of Oskari but as a separate whole. This makes the updating of the Oskari core much easier in the future.
+</Text>
+
+<Text>
+And to make sure that Oskari is a living, developing open source project for years to come for your organisation and everybody else, <b>it is highly recommended that you also contribute to the main Oskari GitHub branch and inform other Oskari users about what you are doing or have done.</b> This way other Oskari users can both enjoy the new functionalities and possibilities of Oskari but also contribute to the further development of the software. Oskari is open source at heart and if you create something Oskari-related, please publish it as open source.
+</Text>
+
+<h3>Documentation</h3>
+
+<Text>
+To make sure that the next developer who either updates your Oskari instance or further develops your new functionalities/configurations knows what to do and what to take into account, you must request proper documentation. 
+</Text>
+
+<Text>
+The documentation should include, for example, what kind of configurations have been done, how, and where. For new functionalities the names of the new bundles (basically modules of Oskari that do the wanted functionality), their events and requests should be listed. The documentation should be so comprehensive that a person who hasn’t been in the original development team can continue the work.
```

Comment:
It's unlikely that application specific code provides their own events and requests, but I guess it could. The implementation of functionalities should be available on the application repository (if not contributed to Oskari repositories). The functionalities that the application uses should be queryable from the database and based on the application it might not be trivial to document these. However anyndocumentation would of course benefit anyone that has to maintain it in the future.

Reply:
Contributing the changes to Oskari or having them as application specifc code also has some difference about what needs to be documented etc. The documentation on Oskari should be generic, but of course application specific documentation can be very clear on what goes where and how does that particular application work.